### ISSUE TITLE
aiida crashes if file for extra info is not saved

### ISSUE THREAD

**Issue by alinelena** [aiidateam/aiida-core#4552]
Trying to add a new computer/code will crash aiida if the file is not saved

```
Traceback (most recent call last):
  File "/srv/vault/anaconda3/envs/aiida/bin/verdi", line 8, in <module>
    sys.exit(verdi())
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/click/core.py", line 1135, in invoke
    sub_ctx = cmd.make_context(cmd_name, args, parent=ctx)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/click/core.py", line 641, in make_context
    self.parse_args(ctx, args)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/click/core.py", line 940, in parse_args
    value, args = param.handle_parse_result(ctx, opts, args)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/click/core.py", line 1476, in handle_parse_result
    value = invoke_param_callback(
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/click/core.py", line 96, in invoke_param_callback
    return callback(ctx, param, value)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/aiida/cmdline/params/options/interactive.py", line 276, in prompt_callback
    value = self.prompt_loop(ctx, param, value)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/aiida/cmdline/params/options/interactive.py", line 210, in simple_prompt_loop
    value = self.prompt_func(ctx)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/aiida/cmdline/params/options/interactive.py", line 311, in prompt_func
    return edit_multiline_template(self.template, extension=self.extension, **kwargs)
  File "/srv/vault/anaconda3/envs/aiida/lib/python3.8/site-packages/aiida/cmdline/utils/multi_line_input.py", line 34, in edit_multiline_template
    return value
UnboundLocalError: local variable 'value' referenced before assignment
```

steps to reproduce, here i use vim

verdi computer setup

fill everything as prompted when asked to do so.
when comes to append extra lines, do not save just use :q  that will generate the crash above.

this happens for both computer/code mainly due to assumption that a file will always exist to be appended

**Comment by chrisjsewell** [comment]
thanks @alinelena

**Comment by greschd** [comment]
@alinelena Which AiiDA version are you using here? There was a similar issue recently, fixed in #4436. The fix should be in version `1.4.3`.

**Comment by sphuber** [comment]
Yes, this was fixed in PR #4436 which was released with `1.4.3`

**Comment by sphuber** [comment]
Duplicate of #4435
